lib/class-wp-rest-block-types-controller.php

```diff
@@ -240,25 +240,26 @@ public function prepare_item_for_response( $block_type, $request ) {
 		$schema       = $this->get_item_schema();
 		$extra_fields = array(
 			'name'          => 'name',
+			'title'         => 'title',
```

Comment:
I ordered them the same way as in WordPress core here:

https://github.com/WordPress/wordpress-develop/blob/822ca9ebc7409abce791e4318626ea0bea75fe88/src/wp-admin/includes/post.php#L2237

to make it easier to compare.